image/Dockerfile

```diff
@@ -94,6 +94,10 @@ FROM emscripten_base
 # Ubuntu steps:
 COPY gehealthcarerootca1.crt gehealthcarerootca2.crt /usr/local/share/ca-certificates
 RUN update-ca-certificates
+
+# Copy the wasm.cmake build configuration file
+COPY wasm.cmake /opt/wasm-deps/share/cmake/
```

Comment:
Is the the `/share/cmake/wasm.cmake` naming convention here a well-established pattern also found elsewhere, or just your best suggestion for a suitable location?

Reply:
Ok. I don't have any better suggestions myself, so it's probably a good place for now.

It would be very nice if we could get CMake to auto-detect the `wasm.cmake` file by placing it in a special folder, but we can look into that later.